kernel/packages/config/index.ts

```diff
@@ -100,10 +100,6 @@ export const RESIZE_SERVICE = qs.RESIZE_SERVICE
 export const HOTSCENES_SERVICE = qs.HOTSCENES_SERVICE
 export const REALM = qs.realm
 
-export const VOICE_CHAT_DISABLED_FLAG = location.search.includes('VOICE_CHAT_DISABLED')
```

Comment:
I'd leave this URL flag. It doesn't hurt anyone.

Reply:
Done!